### ISSUE TITLE
manual/#en/primitives page has an error

### ISSUE THREAD

**Issue by jshji252** [mrdoob/three.js#25372]
### Description

Hello!

Thanks for three.js!

I think the official web page's [primitives](https://threejs.org/manual/#en/primitives) page has a bug:
```
Uncaught TypeError: Cannot read properties of null (reading '1')
    at makeExample (threejs-primitives.js:906:23)
    at createPrimitiveDOM (threejs-primitives.js:930:3)
    at NodeList.forEach (<anonymous>)
    at threejs-primitives.js:1197:50
```

Looks like regex is returning some empty string?

I'm on M1 macbook pro, and tried Chrome, Safari, and Firefox.

### Reproduction steps

1. Go to the web page : https://threejs.org/manual/#en/primitives

### Version

r149

### Device

Desktop

### Browser

Chrome, Firefox, Safari

### OS

MacOS

**Comment by Mugen87** [comment]
Unfortunately, #25312 broke this.

@LeviPesin The new formatting does not work with the regular expression used in `threejs-primitives` and potentially other pages. 

BTW: That's why I'm not a fan of automatic lint fixing. It's important to still check each example/page for breakage because of issues like the one above.

**Comment by jshji252** [comment]
Hey @Mugen87 thank you for quick reply!

If your time allows, could you help me understand how auto formatting can break regex?

I actually wanted to fix this bug myself and cloned the repo to carefully compare the newest vs previous code and it really was just change in formatting, so I gave up and filed an issue. If I can at least learn something from this, then i could call my 60 minutes of unfruitful debugging session was worthwhile :). 

Thank you!

**Comment by Mugen87** [comment]
When a regex expect indentation to be of a certain type like soft tabs, you can't change the formatting to hard tabs without updating the regex.

**Comment by jshji252** [comment]
@Mugen87 thank you so much for your reply!

Yeah your example rings a bell, i've seen exactly that change in the source code.

Hope it gets fixed soon, thank you again :)

**Comment by Mugen87** [comment]
Fixed via dd998eb3c842d90e367b14aaf191e068cb804cf9.
